Eventing infinite recursion 7.1 (#2882)
couchbase/docs-server: modules/eventing/pages/troubleshooting-best-practices.adoc

Eventing Functions can trigger data mutations. To avoid a cyclic generation of data changes, ensure that you carefully consider the below aspects while specifying source and destination collections:

=== Infinite recursion protection

* Avoid infinite recursions.

> If you are using a series of Eventing Functions, then ensure that destination collections to which Eventing Functions perform a write operation, do not have other Eventing Functions configured that can create a loop by triggering cyclic mutations. For example the following design demonstrates an infinite recursion:

Reviewer: ```suggestion
If you are using a series of Eventing Functions, ensure they do not cause cyclic mutations by triggering write operations that, in turn, trigger other Eventing Functions. 
For example the following design demonstrates an infinite recursion:
```
